Flaky LineTcpReceiverTest.testRenameTableSameMeta

### Describe the bug

```
2023-06-16T09:49:46.552139Z I i.q.c.m.EngineMigration upgrading database [version=427]
2023-06-16T09:49:46.552196Z I i.q.c.m.EngineMigration upgraded tables to 427
2023-06-16T09:49:46.552365Z I i.q.c.TableNameRegistryFileStore reloading tables file [path=/tmp/junit13246187388419942853/dbRoot/tables.d.0]
2023-06-16T09:49:46.557038Z I i.q.t.AbstractTest Starting test LineTcpReceiverTest#testRenameTableSameMeta[WITH_WAL]
2023-06-16T09:49:46.557438Z I i.q.c.TableNameRegistryFileStore reloading tables file [path=/tmp/junit13246187388419942853/dbRoot/tables.d.0]
2023-06-16T09:49:46.557625Z A IODispatcher listening on 0.0.0.0:9002 [fd=19 backlog=64]
2023-06-16T09:49:46.557889Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest worker pool started [pool=testing]
2023-06-16T09:49:46.558895Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest os scheduled worker started [name=questdb-testing-26]
2023-06-16T09:49:46.558983Z I IODispatcher connected [ip=127.0.0.1, fd=21]
2023-06-16T09:49:46.559230Z I i.q.c.CairoEngine locked [table=`weather~1`, thread=90]
2023-06-16T09:49:46.570788Z I i.q.c.p.WriterPool created [table=`weather~1`, thread=90]
2023-06-16T09:49:46.570852Z I i.q.c.TableWriter open 'weather'
2023-06-16T09:49:46.571519Z I i.q.c.CairoEngine unlocked [table=`weather`]
2023-06-16T09:49:46.572062Z I i.q.c.l.t.LineTcpMeasurementScheduler created table [tableName=weather]
2023-06-16T09:49:46.572082Z I i.q.c.p.WalWriterPool open 'weather~1' [at=0:0]
2023-06-16T09:49:46.572100Z I i.q.c.w.WalWriter open 'weather~1'
2023-06-16T09:49:46.572451Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/0/_event.i']
2023-06-16T09:49:46.572593Z I i.q.c.p.MetadataPool open 'weather~1' [at=0:0]
2023-06-16T09:49:46.576454Z I i.q.c.p.WalWriterPool open 'weather~1' [at=0:1]
2023-06-16T09:49:46.576476Z I i.q.c.w.WalWriter open 'weather~1'
2023-06-16T09:49:46.576848Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~1/wal2/0/_event.i']
2023-06-16T09:49:46.578763Z I i.q.c.CairoEngine renamed table [from='weather', to='meteorology', wal=2', seqTxn=2]
2023-06-16T09:49:46.578782Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
2023-06-16T09:49:46.580174Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/1/_event.i']
2023-06-16T09:49:46.580696Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=weather]
2023-06-16T09:49:46.580702Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=weather]
2023-06-16T09:49:46.580719Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
2023-06-16T09:49:46.580839Z I i.q.c.CairoEngine locked [table=`weather~2`, thread=90]
2023-06-16T09:49:46.595755Z I i.q.c.p.WriterPool created [table=`weather~2`, thread=90]
2023-06-16T09:49:46.595808Z I i.q.c.TableWriter open 'weather'
2023-06-16T09:49:46.596482Z I i.q.c.CairoEngine unlocked [table=`weather`]
2023-06-16T09:49:46.596760Z I i.q.c.l.t.LineTcpMeasurementScheduler created table [tableName=weather]
2023-06-16T09:49:46.596781Z I i.q.c.p.WalWriterPool open 'weather~2' [at=0:0]
2023-06-16T09:49:46.596800Z I i.q.c.w.WalWriter open 'weather~2'
2023-06-16T09:49:46.597129Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~2/wal1/0/_event.i']
2023-06-16T09:49:46.597270Z I i.q.c.p.MetadataPool open 'weather~2' [at=0:0]
2023-06-16T09:49:46.598387Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~2/wal1/1/_event.i']
2023-06-16T09:49:46.598851Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/2/_event.i']
2023-06-16T09:49:46.598962Z I IODispatcher scheduling disconnect [fd=21, reason=0]
2023-06-16T09:49:46.598968Z I IODispatcher disconnected [ip=127.0.0.1, fd=21, src=queue]
2023-06-16T09:49:46.599026Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=meteorology]
2023-06-16T09:49:46.599030Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=meteorology]
2023-06-16T09:49:46.613185Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
2023-06-16T09:49:46.615311Z I i.q.c.p.WriterPool >> [table=`weather~1`, thread=89]
2023-06-16T09:49:46.615619Z I i.q.c.TableWriter processing WAL [path=/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/0, roLo=0, roHi=2, seqTxn=1, tsMin=2016-06-13T17:43:50.100400Z, tsMax=2016-06-13T17:43:50.100500Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
2023-06-16T09:49:46.615802Z I i.q.c.TableWriter switched partition [path='/tmp/junit13246187388419942853/dbRoot/weather~1/2016-06-13']
2023-06-16T09:49:46.616133Z I i.q.c.TableWriter removed [path=/tmp/junit13246187388419942853/dbRoot/weather~1/2016-06-13]
2023-06-16T09:49:46.616154Z I i.q.c.TableWriter o3 partition task [table=meteorology, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=0, o3TimestampMin=2016-06-13T17:43:50.100400Z, o3Timestamp=2016-06-13T17:43:50.100400Z, o3TimestampMax=2016-06-13T17:43:50.100500Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=-1, partitionSize=2, maxTimestamp=, last=false, append=false, pCount=1, flattenTimestamp=false, memUsed=19150159]
2023-06-16T09:49:46.616558Z I i.q.c.TableWriter switched partition [path='/tmp/junit13246187388419942853/dbRoot/weather~1/2016-06-13']
2023-06-16T09:49:46.616906Z I i.q.c.TableWriter processing WAL [path=/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/1, roLo=0, roHi=2, seqTxn=3, tsMin=2016-06-13T17:43:50.100600Z, tsMax=2016-06-13T17:43:50.100700Z, commitToTimestamp=2016-06-13T17:43:50.101000Z]
2023-06-16T09:49:46.617004Z I i.q.c.TableWriter o3 partition task [table=meteorology, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=2, o3TimestampMin=2016-06-13T17:43:50.100600Z, o3Timestamp=2016-06-13T17:43:50.100600Z, o3TimestampMax=2016-06-13T17:43:50.100700Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=0, partitionSize=4, maxTimestamp=2016-06-13T17:43:50.100500Z, last=true, append=true, pCount=1, flattenTimestamp=false, memUsed=31766157]
2023-06-16T09:49:46.617183Z I i.q.c.TableWriter processing WAL [path=/tmp/junit13246187388419942853/dbRoot/weather~1/wal1/2, roLo=0, roHi=1, seqTxn=4, tsMin=2016-06-13T17:43:50.101000Z, tsMax=2016-06-13T17:43:50.101000Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
2023-06-16T09:49:46.617254Z I i.q.c.TableWriter o3 partition task [table=meteorology, partitionIsReadOnly=false, srcOooBatchRowSize=1, srcOooLo=0, srcOooHi=0, srcOooMax=1, o3RowCount=1, o3LagRowCount=0, srcDataMax=4, o3TimestampMin=2016-06-13T17:43:50.101000Z, o3Timestamp=2016-06-13T17:43:50.101000Z, o3TimestampMax=2016-06-13T17:43:50.101000Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=0, partitionSize=5, maxTimestamp=2016-06-13T17:43:50.100700Z, last=true, append=true, pCount=1, flattenTimestamp=false, memUsed=31766113]
2023-06-16T09:49:46.617367Z I i.q.c.w.ApplyWal2TableJob job finished [table=weather~1, seqTxn=4, transactions=3, rows=5, time=1ms, rate=3071rows/s, physicalWrittenRowsMultiplier=1.0]
2023-06-16T09:49:46.617398Z I i.q.c.p.WriterPool << [table=`weather~1`, thread=89]
2023-06-16T09:49:46.617426Z I i.q.c.p.WriterPool >> [table=`weather~2`, thread=89]
2023-06-16T09:49:46.617530Z I i.q.c.TableWriter processing WAL [path=/tmp/junit13246187388419942853/dbRoot/weather~2/wal1/0, roLo=0, roHi=2, seqTxn=1, tsMin=2016-06-13T17:43:50.100800Z, tsMax=2016-06-13T17:43:50.100900Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
2023-06-16T09:49:46.617619Z I i.q.c.TableWriter switched partition [path='/tmp/junit13246187388419942853/dbRoot/weather~2/2016-06-13']
2023-06-16T09:49:46.617826Z I i.q.c.TableWriter removed [path=/tmp/junit13246187388419942853/dbRoot/weather~2/2016-06-13]
2023-06-16T09:49:46.617838Z I i.q.c.TableWriter o3 partition task [table=weather, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=0, o3TimestampMin=2016-06-13T17:43:50.100800Z, o3Timestamp=2016-06-13T17:43:50.100800Z, o3TimestampMax=2016-06-13T17:43:50.100900Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=-1, partitionSize=2, maxTimestamp=, last=false, append=false, pCount=1, flattenTimestamp=false, memUsed=31766011]
2023-06-16T09:49:46.618094Z I i.q.c.TableWriter switched partition [path='/tmp/junit13246187388419942853/dbRoot/weather~2/2016-06-13']
2023-06-16T09:49:46.618219Z I i.q.c.w.ApplyWal2TableJob job finished [table=weather~2, seqTxn=1, transactions=1, rows=2, time=0ms, rate=2828rows/s, physicalWrittenRowsMultiplier=1.0]
2023-06-16T09:49:46.618242Z I i.q.c.p.WriterPool << [table=`weather~2`, thread=89]
2023-06-16T09:49:46.618423Z I i.q.c.p.ReaderPool open 'weather~1' [at=0:0]
2023-06-16T09:49:46.618655Z I i.q.c.TableReader open partition /tmp/junit13246187388419942853/dbRoot/weather~1/2016-06-13 [rowCount=5, partitionNameTxn=-1, transientRowCount=5, partitionIndex=0, partitionCount=1]
2023-06-16T09:49:46.618813Z I i.q.c.p.ReaderPool open 'weather~2' [at=0:0]
2023-06-16T09:49:46.618988Z I i.q.c.TableReader open partition /tmp/junit13246187388419942853/dbRoot/weather~2/2016-06-13 [rowCount=2, partitionNameTxn=-1, transientRowCount=2, partitionIndex=0, partitionCount=1]
2023-06-16T09:49:46.619191Z E i.q.t.c.l.t.AbstractLineTcpReceiverTest Stopping ILP worker pool because of an error
java.lang.AssertionError: expected:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
south7	70.0	2016-06-13T17:43:50.101000Z
> but was:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:351)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:334)
	at io.questdb.test.tools.TestUtils.assertCursor(TestUtils.java:128)
	at io.questdb.test.AbstractCairoTest.assertCursor(AbstractCairoTest.java:551)
	at io.questdb.test.AbstractCairoTest.assertCursorTwoPass(AbstractCairoTest.java:555)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.assertTable(AbstractLineTcpReceiverTest.java:253)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.lambda$testRenameTableSameMeta$27(LineTcpReceiverTest.java:780)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.lambda$runInContext$2(AbstractLineTcpReceiverTest.java:299)
	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$4(AbstractCairoTest.java:285)
	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:540)
	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:282)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.runInContext(AbstractLineTcpReceiverTest.java:291)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta(LineTcpReceiverTest.java:753)
	at jdk.internal.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.lang.Thread.run(Thread.java:833)

2023-06-16T09:49:46.619452Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=weather]
2023-06-16T09:49:46.619457Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=weather]
2023-06-16T09:49:46.619806Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~2:weather
2023-06-16T09:49:46.619876Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest cleaned worker [name=testing, worker=0, total=1]
2023-06-16T09:49:46.619885Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest os scheduled worker stopped [name=questdb-testing-26]
2023-06-16T09:49:46.620348Z I i.q.c.l.t.LineTcpWriterJob line protocol writer closing [threadId=0]
2023-06-16T09:49:46.620409Z I IODispatcher closed
2023-06-16T09:49:46.620413Z E i.q.t.c.l.t.AbstractLineTcpReceiverTest Stopping ILP receiver because of an error
java.lang.AssertionError: expected:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
south7	70.0	2016-06-13T17:43:50.101000Z
> but was:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:351)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:334)
	at io.questdb.test.tools.TestUtils.assertCursor(TestUtils.java:128)
	at io.questdb.test.AbstractCairoTest.assertCursor(AbstractCairoTest.java:551)
	at io.questdb.test.AbstractCairoTest.assertCursorTwoPass(AbstractCairoTest.java:555)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.assertTable(AbstractLineTcpReceiverTest.java:253)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.lambda$testRenameTableSameMeta$27(LineTcpReceiverTest.java:780)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.lambda$runInContext$2(AbstractLineTcpReceiverTest.java:299)
	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$4(AbstractCairoTest.java:285)
	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:540)
	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:282)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.runInContext(AbstractLineTcpReceiverTest.java:291)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta(LineTcpReceiverTest.java:753)
	at jdk.internal.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.lang.Thread.run(Thread.java:833)

2023-06-16T09:49:46.621035Z I i.q.c.p.ReaderPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.621091Z I i.q.c.p.ReaderPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.621560Z I i.q.c.TableWriter closed 'weather'
2023-06-16T09:49:46.621565Z I i.q.c.p.WriterPool closed [table=`weather~2`, reason=POOL_CLOSED, by=89]
2023-06-16T09:49:46.621756Z I i.q.c.TableWriter closed 'meteorology'
2023-06-16T09:49:46.621759Z I i.q.c.p.WriterPool closed [table=`weather~1`, reason=POOL_CLOSED, by=89]
2023-06-16T09:49:46.621808Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=weather~2]
2023-06-16T09:49:46.621834Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=weather~1]
2023-06-16T09:49:46.621841Z I i.q.c.p.MetadataPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.621845Z I i.q.c.p.MetadataPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.621931Z I i.q.c.w.WalWriter closed 'weather'
2023-06-16T09:49:46.621934Z I i.q.c.p.WalWriterPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.622005Z I i.q.c.w.WalWriter closed 'meteorology'
2023-06-16T09:49:46.622006Z I i.q.c.p.WalWriterPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
2023-06-16T09:49:46.622054Z I i.q.c.w.WalWriter closed 'meteorology'
2023-06-16T09:49:46.622056Z I i.q.c.p.WalWriterPool closed 'weather~1' [at=0:1, reason=POOL_CLOSED]
2023-06-16T09:49:46.622104Z I i.q.t.AbstractCairoTest Tearing down test LineTcpReceiverTest#testRenameTableSameMeta[WITH_WAL]
```

### To reproduce

_No response_

### Expected Behavior

_No response_

### Environment

```markdown
- **QuestDB version**:
- **OS**:
- **Browser**:
```


### Additional context

_No response_

With more debug logging:
```
[1] 2023-06-20T09:10:20.049451Z I i.q.t.TestListener >>>> io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta[WITH_WAL]
[7620] 2023-06-20T09:10:20.049802Z I i.q.t.AbstractTest Starting test LineTcpReceiverTest#testRenameTableSameMeta[WITH_WAL]
[7620] 2023-06-20T09:10:20.049958Z I i.q.t.s.TestFilesFacadeImpl cannot remove file: /tmp/junit9693785222249370873/dbRoot/tables.d.0, errno:2
[7620] 2023-06-20T09:10:20.050151Z I i.q.c.TableNameRegistryFileStore reloading tables file [path=/tmp/junit9693785222249370873/dbRoot/tables.d.0]
[7620] 2023-06-20T09:10:20.050203Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (A)
[7620] 2023-06-20T09:10:20.050416Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (B)
[7620] 2023-06-20T09:10:20.050796Z A IODispatcher listening on 0.0.0.0:9002 [fd=22 backlog=64]
[7620] 2023-06-20T09:10:20.051034Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest worker pool started [pool=testing]
[7620] 2023-06-20T09:10:20.051047Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (C)
[7620] 2023-06-20T09:10:20.051404Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (D)
[7620] 2023-06-20T09:10:20.051410Z I i.q.t.c.l.t.LineTcpReceiverTest sendWaitWalReleaseCount :: (A) lineData=weather,location=west1 temperature=10 1465839830100400200
weather,location=west2 temperature=20 1465839830100500200
weather,location=east3 temperature=30 1465839830100600200
weather,location=west4 temperature=40 1465839830100700200
weather,location=west5 temperature=50 1465839830100800200
weather,location=east6 temperature=60 1465839830100900200
weather,location=south7 temperature=70 1465839830101000200
meteorology,location=south8 temperature=80 1465839830101000200
[7620] 2023-06-20T09:10:20.051418Z I i.q.t.c.l.t.LineTcpReceiverTest sendWaitWalReleaseCount :: (B)
[7620] 2023-06-20T09:10:20.051576Z I i.q.t.c.l.t.LineTcpReceiverTest sendWaitWalReleaseCount :: (C)
[7621] 2023-06-20T09:10:20.051627Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest os scheduled worker started [name=questdb-testing-5015]
[7621] 2023-06-20T09:10:20.051675Z I IODispatcher connected [ip=127.0.0.1, fd=24]
[7621] 2023-06-20T09:10:20.051873Z I i.q.c.CairoEngine locked [table=`weather~1`, thread=7621]
[7621] 2023-06-20T09:10:20.051969Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_meta
[7621] 2023-06-20T09:10:20.053582Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.o
[7621] 2023-06-20T09:10:20.054294Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.k
[7621] 2023-06-20T09:10:20.055879Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_txn
[7621] 2023-06-20T09:10:20.057161Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_cv
[7621] 2023-06-20T09:10:20.059320Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_todo_
[7621] 2023-06-20T09:10:20.060042Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_name
[7621] 2023-06-20T09:10:20.061118Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_meta.0
[7621] 2023-06-20T09:10:20.061939Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_meta
[7621] 2023-06-20T09:10:20.062547Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_wal_index.d
[7621] 2023-06-20T09:10:20.062724Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_meta
[7621] 2023-06-20T09:10:20.062788Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_txnlog
[7621] 2023-06-20T09:10:20.062827Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_txnlog.meta.d
[7621] 2023-06-20T09:10:20.062858Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/txn_seq/_txnlog.meta.i
[7621] 2023-06-20T09:10:20.062927Z I i.q.c.p.WriterPool created [table=`weather~1`, thread=7621]
[7621] 2023-06-20T09:10:20.062946Z I i.q.c.TableWriter open 'weather'
[7621] 2023-06-20T09:10:20.062967Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_todo_
[7621] 2023-06-20T09:10:20.063030Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_cv
[7621] 2023-06-20T09:10:20.063093Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_txn
[7621] 2023-06-20T09:10:20.063315Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.o
[7621] 2023-06-20T09:10:20.063401Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.k
[7621] 2023-06-20T09:10:20.063477Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.v
[7621] 2023-06-20T09:10:20.063594Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/location.c
[7621] 2023-06-20T09:10:20.064035Z I i.q.c.CairoEngine unlocked [table=`weather`]
[7621] 2023-06-20T09:10:20.064514Z I i.q.c.l.t.LineTcpMeasurementScheduler created table [tableName=weather]
[7621] 2023-06-20T09:10:20.064526Z I i.q.c.p.WalWriterPool open 'weather~1' [at=0:0]
[7621] 2023-06-20T09:10:20.064534Z I i.q.c.w.WalWriter open 'weather~1'
[7621] 2023-06-20T09:10:20.064565Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1.lock
[7621] 2023-06-20T09:10:20.064694Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0.lock
[7621] 2023-06-20T09:10:20.064765Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/location.d
[7621] 2023-06-20T09:10:20.064787Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/temperature.d
[7621] 2023-06-20T09:10:20.064816Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/timestamp.d
[7621] 2023-06-20T09:10:20.064840Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/_meta
[7621] 2023-06-20T09:10:20.064900Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/_event
[7621] 2023-06-20T09:10:20.064948Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/_event.i
[7621] 2023-06-20T09:10:20.064982Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0/_event.i']
[7621] 2023-06-20T09:10:20.065165Z I i.q.c.p.MetadataPool open 'weather~1' [at=0:0]
[7621] 2023-06-20T09:10:20.066074Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1.lock
[7621] 2023-06-20T09:10:20.066092Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (B)
[7621] 2023-06-20T09:10:20.068441Z I i.q.c.p.WriterPool >> [table=`weather~1`, thread=7621]
[7621] 2023-06-20T09:10:20.068705Z I i.q.c.TableWriter processing WAL [path=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/0, roLo=0, roHi=2, seqTxn=1, tsMin=2016-06-13T17:43:50.100400Z, tsMax=2016-06-13T17:43:50.100500Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
[7621] 2023-06-20T09:10:20.068771Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/location.d
[7621] 2023-06-20T09:10:20.068822Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/temperature.d
[7621] 2023-06-20T09:10:20.068841Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/timestamp.d
[7621] 2023-06-20T09:10:20.068857Z I i.q.c.TableWriter switched partition [path='/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13']
[7621] 2023-06-20T09:10:20.069155Z I i.q.c.TableWriter removed [path=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13]
[7621] 2023-06-20T09:10:20.069163Z I i.q.c.TableWriter o3 partition task [table=weather, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=0, o3TimestampMin=2016-06-13T17:43:50.100400Z, o3Timestamp=2016-06-13T17:43:50.100400Z, o3TimestampMax=2016-06-13T17:43:50.100500Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=-1, partitionSize=2, maxTimestamp=, last=false, append=false, pCount=1, flattenTimestamp=false, memUsed=43044957]
[7621] 2023-06-20T09:10:20.069256Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/location.d
[7621] 2023-06-20T09:10:20.069296Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/temperature.d
[7621] 2023-06-20T09:10:20.069323Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/timestamp.d
[7621] 2023-06-20T09:10:20.069499Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/location.d
[7621] 2023-06-20T09:10:20.069540Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/temperature.d
[7621] 2023-06-20T09:10:20.069551Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13/timestamp.d
[7621] 2023-06-20T09:10:20.069561Z I i.q.c.TableWriter switched partition [path='/tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13']
[7621] 2023-06-20T09:10:20.069721Z I i.q.c.w.ApplyWal2TableJob job finished [table=weather~1, seqTxn=1, transactions=1, rows=2, time=1ms, rate=1941rows/s, physicalWrittenRowsMultiplier=1.0]
[7621] 2023-06-20T09:10:20.069752Z I i.q.c.p.WriterPool << [table=`weather~1`, thread=7621]
[7621] 2023-06-20T09:10:20.069851Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (C)
[7621] 2023-06-20T09:10:20.069929Z I i.q.c.p.WalWriterPool open 'weather~1' [at=0:1]
[7621] 2023-06-20T09:10:20.069934Z I i.q.c.w.WalWriter open 'weather~1'
[7621] 2023-06-20T09:10:20.069944Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2.lock
[7621] 2023-06-20T09:10:20.070019Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0.lock
[7621] 2023-06-20T09:10:20.070068Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/location.d
[7621] 2023-06-20T09:10:20.070088Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/temperature.d
[7621] 2023-06-20T09:10:20.070104Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/timestamp.d
[7621] 2023-06-20T09:10:20.070125Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/_meta
[7621] 2023-06-20T09:10:20.070183Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/_event
[7621] 2023-06-20T09:10:20.070215Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/_event.i
[7621] 2023-06-20T09:10:20.070246Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~1/wal2/0/_event.i']
[7621] 2023-06-20T09:10:20.072488Z I i.q.c.CairoEngine renamed table [from='weather', to='meteorology', wal=2', seqTxn=2]
[7621] 2023-06-20T09:10:20.072530Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
[7621] 2023-06-20T09:10:20.072546Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_name
[7621] 2023-06-20T09:10:20.073391Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (D)
[7621] 2023-06-20T09:10:20.073590Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/location.d
[7621] 2023-06-20T09:10:20.073650Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/temperature.d
[7621] 2023-06-20T09:10:20.073694Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/timestamp.d
[7621] 2023-06-20T09:10:20.073792Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/_meta
[7621] 2023-06-20T09:10:20.074010Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/_event
[7621] 2023-06-20T09:10:20.074052Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/_event.i
[7621] 2023-06-20T09:10:20.074083Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1/_event.i']
[7621] 2023-06-20T09:10:20.074509Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=weather]
[7621] 2023-06-20T09:10:20.074511Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=weather]
[7621] 2023-06-20T09:10:20.074535Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
[7621] 2023-06-20T09:10:20.074708Z I i.q.c.CairoEngine locked [table=`weather~2`, thread=7621]
[7621] 2023-06-20T09:10:20.074768Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_meta
[7621] 2023-06-20T09:10:20.075553Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.o
[7621] 2023-06-20T09:10:20.076068Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.k
[7621] 2023-06-20T09:10:20.076557Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_txn
[7621] 2023-06-20T09:10:20.076993Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_cv
[7621] 2023-06-20T09:10:20.077348Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_todo_
[7621] 2023-06-20T09:10:20.077718Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_name
[7621] 2023-06-20T09:10:20.078353Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_meta.0
[7621] 2023-06-20T09:10:20.078815Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_meta
[7621] 2023-06-20T09:10:20.079245Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_wal_index.d
[7621] 2023-06-20T09:10:20.079335Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_meta
[7621] 2023-06-20T09:10:20.079374Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_txnlog
[7621] 2023-06-20T09:10:20.079409Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_txnlog.meta.d
[7621] 2023-06-20T09:10:20.079439Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/txn_seq/_txnlog.meta.i
[7621] 2023-06-20T09:10:20.079491Z I i.q.c.p.WriterPool created [table=`weather~2`, thread=7621]
[7621] 2023-06-20T09:10:20.079508Z I i.q.c.TableWriter open 'weather'
[7621] 2023-06-20T09:10:20.079526Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_todo_
[7621] 2023-06-20T09:10:20.079547Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_cv
[7621] 2023-06-20T09:10:20.079596Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/_txn
[7621] 2023-06-20T09:10:20.079681Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.o
[7621] 2023-06-20T09:10:20.079704Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.k
[7621] 2023-06-20T09:10:20.079722Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.v
[7621] 2023-06-20T09:10:20.079748Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/location.c
[7621] 2023-06-20T09:10:20.079916Z I i.q.c.CairoEngine unlocked [table=`weather`]
[7621] 2023-06-20T09:10:20.080004Z I i.q.c.l.t.LineTcpMeasurementScheduler created table [tableName=weather]
[7621] 2023-06-20T09:10:20.080012Z I i.q.c.p.WalWriterPool open 'weather~2' [at=0:0]
[7621] 2023-06-20T09:10:20.080017Z I i.q.c.w.WalWriter open 'weather~2'
[7621] 2023-06-20T09:10:20.080037Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1.lock
[7621] 2023-06-20T09:10:20.086417Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0.lock
[7621] 2023-06-20T09:10:20.086510Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/location.d
[7621] 2023-06-20T09:10:20.086533Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/temperature.d
[7621] 2023-06-20T09:10:20.086552Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/timestamp.d
[7621] 2023-06-20T09:10:20.086575Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/_meta
[7621] 2023-06-20T09:10:20.086628Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/_event
[7621] 2023-06-20T09:10:20.086661Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/_event.i
[7621] 2023-06-20T09:10:20.086692Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0/_event.i']
[7621] 2023-06-20T09:10:20.086814Z I i.q.c.p.MetadataPool open 'weather~2' [at=0:0]
[7621] 2023-06-20T09:10:20.088366Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1.lock
[7621] 2023-06-20T09:10:20.088512Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/location.d
[7621] 2023-06-20T09:10:20.088586Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/temperature.d
[7621] 2023-06-20T09:10:20.088648Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/timestamp.d
[7621] 2023-06-20T09:10:20.088707Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/_meta
[7621] 2023-06-20T09:10:20.088811Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/_event
[7621] 2023-06-20T09:10:20.088890Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/_event.i
[7621] 2023-06-20T09:10:20.088922Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/1/_event.i']
[7621] 2023-06-20T09:10:20.089583Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2.lock
[7621] 2023-06-20T09:10:20.089726Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/location.d
[7621] 2023-06-20T09:10:20.089795Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/temperature.d
[7621] 2023-06-20T09:10:20.089926Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/timestamp.d
[7621] 2023-06-20T09:10:20.090547Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/_meta
[7621] 2023-06-20T09:10:20.090670Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/_event
[7621] 2023-06-20T09:10:20.090755Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/_event.i
[7621] 2023-06-20T09:10:20.090792Z I i.q.c.w.WalWriter opened WAL segment [path='/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2/_event.i']
[7621] 2023-06-20T09:10:20.090915Z I IODispatcher scheduling disconnect [fd=24, reason=0]
[7621] 2023-06-20T09:10:20.090918Z I IODispatcher disconnected [ip=127.0.0.1, fd=24, src=queue]
[7621] 2023-06-20T09:10:20.090988Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=meteorology]
[7621] 2023-06-20T09:10:20.090991Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=meteorology]
[7621] 2023-06-20T09:10:20.091266Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~1:meteorology
[7620] 2023-06-20T09:10:20.091285Z I i.q.t.c.l.t.LineTcpReceiverTest sendWaitWalReleaseCount :: (D)
[7620] 2023-06-20T09:10:20.091287Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (E)
[7620] 2023-06-20T09:10:20.093344Z I i.q.c.p.WriterPool >> [table=`weather~1`, thread=7620]
[7620] 2023-06-20T09:10:20.093686Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~1/_name
[7620] 2023-06-20T09:10:20.093820Z I i.q.c.TableWriter processing WAL [path=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/1, roLo=0, roHi=2, seqTxn=3, tsMin=2016-06-13T17:43:50.100600Z, tsMax=2016-06-13T17:43:50.100700Z, commitToTimestamp=2016-06-13T17:43:50.101000Z]
[7620] 2023-06-20T09:10:20.093898Z I i.q.c.TableWriter o3 partition task [table=meteorology, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=2, o3TimestampMin=2016-06-13T17:43:50.100600Z, o3Timestamp=2016-06-13T17:43:50.100600Z, o3TimestampMax=2016-06-13T17:43:50.100700Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=0, partitionSize=4, maxTimestamp=2016-06-13T17:43:50.100500Z, last=true, append=true, pCount=1, flattenTimestamp=false, memUsed=59955423]
[7620] 2023-06-20T09:10:20.094105Z I i.q.c.TableWriter processing WAL [path=/tmp/junit9693785222249370873/dbRoot/weather~1/wal1/2, roLo=0, roHi=1, seqTxn=4, tsMin=2016-06-13T17:43:50.101000Z, tsMax=2016-06-13T17:43:50.101000Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
[7620] 2023-06-20T09:10:20.094161Z I i.q.c.TableWriter o3 partition task [table=meteorology, partitionIsReadOnly=false, srcOooBatchRowSize=1, srcOooLo=0, srcOooHi=0, srcOooMax=1, o3RowCount=1, o3LagRowCount=0, srcDataMax=4, o3TimestampMin=2016-06-13T17:43:50.101000Z, o3Timestamp=2016-06-13T17:43:50.101000Z, o3TimestampMax=2016-06-13T17:43:50.101000Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=0, partitionSize=5, maxTimestamp=2016-06-13T17:43:50.100700Z, last=true, append=true, pCount=1, flattenTimestamp=false, memUsed=59955379]
[7620] 2023-06-20T09:10:20.094264Z I i.q.c.w.ApplyWal2TableJob job finished [table=weather~1, seqTxn=4, transactions=2, rows=3, time=0ms, rate=6276rows/s, physicalWrittenRowsMultiplier=1.0]
[7620] 2023-06-20T09:10:20.094297Z I i.q.c.p.WriterPool << [table=`weather~1`, thread=7620]
[7620] 2023-06-20T09:10:20.094313Z I i.q.c.p.WriterPool >> [table=`weather~2`, thread=7620]
[7620] 2023-06-20T09:10:20.094417Z I i.q.c.TableWriter processing WAL [path=/tmp/junit9693785222249370873/dbRoot/weather~2/wal1/0, roLo=0, roHi=2, seqTxn=1, tsMin=2016-06-13T17:43:50.100800Z, tsMax=2016-06-13T17:43:50.100900Z, commitToTimestamp=294247-01-10T04:00:54.775807Z]
[7620] 2023-06-20T09:10:20.094473Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/location.d
[7620] 2023-06-20T09:10:20.094500Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/temperature.d
[7620] 2023-06-20T09:10:20.094520Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/timestamp.d
[7620] 2023-06-20T09:10:20.094539Z I i.q.c.TableWriter switched partition [path='/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13']
[7620] 2023-06-20T09:10:20.094775Z I i.q.c.TableWriter removed [path=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13]
[7620] 2023-06-20T09:10:20.094783Z I i.q.c.TableWriter o3 partition task [table=weather, partitionIsReadOnly=false, srcOooBatchRowSize=2, srcOooLo=0, srcOooHi=1, srcOooMax=2, o3RowCount=2, o3LagRowCount=0, srcDataMax=0, o3TimestampMin=2016-06-13T17:43:50.100800Z, o3Timestamp=2016-06-13T17:43:50.100800Z, o3TimestampMax=2016-06-13T17:43:50.100900Z, partitionTimestamp=2016-06-13T00:00:00.000000Z, partitionIndex=-1, partitionSize=2, maxTimestamp=, last=false, append=false, pCount=1, flattenTimestamp=false, memUsed=59955277]
[7620] 2023-06-20T09:10:20.094861Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/location.d
[7620] 2023-06-20T09:10:20.094902Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/temperature.d
[7620] 2023-06-20T09:10:20.094931Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/timestamp.d
[7620] 2023-06-20T09:10:20.095058Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/location.d
[7620] 2023-06-20T09:10:20.095076Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/temperature.d
[7620] 2023-06-20T09:10:20.095089Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta.openRW :: (A) name=/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13/timestamp.d
[7620] 2023-06-20T09:10:20.095101Z I i.q.c.TableWriter switched partition [path='/tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13']
[7620] 2023-06-20T09:10:20.095228Z I i.q.c.w.ApplyWal2TableJob job finished [table=weather~2, seqTxn=1, transactions=1, rows=2, time=0ms, rate=2386rows/s, physicalWrittenRowsMultiplier=1.0]
[7620] 2023-06-20T09:10:20.095247Z I i.q.c.p.WriterPool << [table=`weather~2`, thread=7620]
[7620] 2023-06-20T09:10:20.095382Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (F)
[7620] 2023-06-20T09:10:20.095400Z I i.q.c.p.ReaderPool open 'weather~1' [at=0:0]
[7620] 2023-06-20T09:10:20.095619Z I i.q.c.TableReader open partition /tmp/junit9693785222249370873/dbRoot/weather~1/2016-06-13 [rowCount=5, partitionNameTxn=-1, transientRowCount=5, partitionIndex=0, partitionCount=1]
[7620] 2023-06-20T09:10:20.095775Z I i.q.t.c.l.t.LineTcpReceiverTest testRenameTableSameMeta :: (G)
[7620] 2023-06-20T09:10:20.095787Z I i.q.c.p.ReaderPool open 'weather~2' [at=0:0]
[7620] 2023-06-20T09:10:20.095965Z I i.q.c.TableReader open partition /tmp/junit9693785222249370873/dbRoot/weather~2/2016-06-13 [rowCount=2, partitionNameTxn=-1, transientRowCount=2, partitionIndex=0, partitionCount=1]
[7620] 2023-06-20T09:10:20.096136Z E i.q.t.c.l.t.AbstractLineTcpReceiverTest Stopping ILP worker pool because of an error
java.lang.AssertionError: expected:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
south7	70.0	2016-06-13T17:43:50.101000Z
> but was:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:351)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:334)
	at io.questdb.test.tools.TestUtils.assertCursor(TestUtils.java:128)
	at io.questdb.test.AbstractCairoTest.assertCursor(AbstractCairoTest.java:551)
	at io.questdb.test.AbstractCairoTest.assertCursorTwoPass(AbstractCairoTest.java:555)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.assertTable(AbstractLineTcpReceiverTest.java:253)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.lambda$testRenameTableSameMeta$27(LineTcpReceiverTest.java:802)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.lambda$runInContext$2(AbstractLineTcpReceiverTest.java:299)
	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$4(AbstractCairoTest.java:285)
	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:540)
	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:282)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.runInContext(AbstractLineTcpReceiverTest.java:291)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta(LineTcpReceiverTest.java:762)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.lang.Thread.run(Thread.java:829)
[7621] 2023-06-20T09:10:20.096427Z I i.q.c.l.t.TableUpdateDetails closing table writer [tableName=weather]
[7621] 2023-06-20T09:10:20.096430Z I i.q.c.l.t.TableUpdateDetails closing table parsers [tableName=weather]
[7621] 2023-06-20T09:10:20.097095Z I i.q.t.c.l.t.LineTcpReceiverTest === released WAL writer === weather~2:weather
[7621] 2023-06-20T09:10:20.097137Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest cleaned worker [name=testing, worker=0, total=1]
[7621] 2023-06-20T09:10:20.097144Z I i.q.t.c.l.t.AbstractLineTcpReceiverTest os scheduled worker stopped [name=questdb-testing-5015]
[7620] 2023-06-20T09:10:20.097170Z I i.q.c.l.t.LineTcpWriterJob line protocol writer closing [threadId=0]
[7620] 2023-06-20T09:10:20.097226Z I IODispatcher closed
[7620] 2023-06-20T09:10:20.097231Z E i.q.t.c.l.t.AbstractLineTcpReceiverTest Stopping ILP receiver because of an error
java.lang.AssertionError: expected:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
south7	70.0	2016-06-13T17:43:50.101000Z
> but was:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:351)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:334)
	at io.questdb.test.tools.TestUtils.assertCursor(TestUtils.java:128)
	at io.questdb.test.AbstractCairoTest.assertCursor(AbstractCairoTest.java:551)
	at io.questdb.test.AbstractCairoTest.assertCursorTwoPass(AbstractCairoTest.java:555)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.assertTable(AbstractLineTcpReceiverTest.java:253)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.lambda$testRenameTableSameMeta$27(LineTcpReceiverTest.java:802)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.lambda$runInContext$2(AbstractLineTcpReceiverTest.java:299)
	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$4(AbstractCairoTest.java:285)
	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:540)
	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:282)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.runInContext(AbstractLineTcpReceiverTest.java:291)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta(LineTcpReceiverTest.java:762)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.lang.Thread.run(Thread.java:829)
[7620] 2023-06-20T09:10:20.097455Z I i.q.c.p.ReaderPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.097583Z I i.q.c.p.ReaderPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.097894Z I i.q.c.TableWriter closed 'weather'
[7620] 2023-06-20T09:10:20.097900Z I i.q.c.p.WriterPool closed [table=`weather~2`, reason=POOL_CLOSED, by=7620]
[7620] 2023-06-20T09:10:20.098189Z I i.q.c.TableWriter closed 'meteorology'
[7620] 2023-06-20T09:10:20.098190Z I i.q.c.p.WriterPool closed [table=`weather~1`, reason=POOL_CLOSED, by=7620]
[7620] 2023-06-20T09:10:20.098258Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=weather~2]
[7620] 2023-06-20T09:10:20.098338Z I i.q.c.w.s.TableSequencerAPI releasing idle table sequencer [tableDir=weather~1]
[7620] 2023-06-20T09:10:20.098346Z I i.q.c.p.MetadataPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.098466Z I i.q.c.p.MetadataPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.098618Z I i.q.c.w.WalWriter closed 'weather'
[7620] 2023-06-20T09:10:20.098621Z I i.q.c.p.WalWriterPool closed 'weather~2' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.098767Z I i.q.c.w.WalWriter closed 'meteorology'
[7620] 2023-06-20T09:10:20.098769Z I i.q.c.p.WalWriterPool closed 'weather~1' [at=0:0, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.098909Z I i.q.c.w.WalWriter closed 'meteorology'
[7620] 2023-06-20T09:10:20.098911Z I i.q.c.p.WalWriterPool closed 'weather~1' [at=0:1, reason=POOL_CLOSED]
[7620] 2023-06-20T09:10:20.099000Z I i.q.t.AbstractCairoTest Tearing down test LineTcpReceiverTest#testRenameTableSameMeta[WITH_WAL]
[1] 2023-06-20T09:10:20.117996Z E i.q.t.TestListener ***** Test Failed *****io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta[WITH_WAL] duration_ms=69 : 
java.lang.AssertionError: expected:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
south7	70.0	2016-06-13T17:43:50.101000Z
> but was:<location	temperature	timestamp
west5	50.0	2016-06-13T17:43:50.100800Z
east6	60.0	2016-06-13T17:43:50.100900Z
>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:120)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:351)
	at io.questdb.test.tools.TestUtils.assertEquals(TestUtils.java:334)
	at io.questdb.test.tools.TestUtils.assertCursor(TestUtils.java:128)
	at io.questdb.test.AbstractCairoTest.assertCursor(AbstractCairoTest.java:551)
	at io.questdb.test.AbstractCairoTest.assertCursorTwoPass(AbstractCairoTest.java:555)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.assertTable(AbstractLineTcpReceiverTest.java:253)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.lambda$testRenameTableSameMeta$27(LineTcpReceiverTest.java:802)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.lambda$runInContext$2(AbstractLineTcpReceiverTest.java:299)
	at io.questdb.test.AbstractCairoTest.lambda$assertMemoryLeak$4(AbstractCairoTest.java:285)
	at io.questdb.test.tools.TestUtils.assertMemoryLeak(TestUtils.java:540)
	at io.questdb.test.AbstractCairoTest.assertMemoryLeak(AbstractCairoTest.java:282)
	at io.questdb.test.cutlass.line.tcp.AbstractLineTcpReceiverTest.runInContext(AbstractLineTcpReceiverTest.java:291)
	at io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta(LineTcpReceiverTest.java:762)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.lang.Thread.run(Thread.java:829)
[1] 2023-06-20T09:10:20.118077Z I i.q.t.TestListener <<<< io.questdb.test.cutlass.line.tcp.LineTcpReceiverTest.testRenameTableSameMeta[WITH_WAL] duration_ms=69
```

This was in CI. Never managed to reproduce it locally.

This logging comes from this commit: https://github.com/questdb/questdb/commit/7bbbc007ec491486bb16329a5f71e6ab3ae299aa

--- a/core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpReceiverTest.java
+++ b/core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpReceiverTest.java
@@ -79,6 +79,7 @@ import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
 
 import static io.questdb.cutlass.auth.AuthUtils.EC_ALGORITHM;
+import static io.questdb.test.tools.TestUtils.assertEventually;
 
 @RunWith(Parameterized.class)
 public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
@@ -205,7 +206,7 @@ public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
             new Thread(() -> {
                 try (Socket socket = getSocket()) {
                     for (int i = 0; i < numOfRows; i++) {
-                        String value = (i % 2 == 0) ? "\"test" + i + "\"" : "" + i;
+                        String value = (i % 2 == 0) ? "\"test" + i + "\"" : String.valueOf(i);
                         sendToSocket(socket, tableName + ",abcdef=x col=" + value + "\n");
                     }
                 } catch (Exception e) {
@@ -745,6 +746,7 @@ public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
                                 --count == 0
                 ) {
                     renameTable(weather, meteorology);
+                    Os.sleep(100);
                 }
                 return super.openRW(name, opts);
             }
@@ -762,22 +764,35 @@ public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
 
             sendWaitWalReleaseCount(lineData, 3);
 
-            mayDrainWalQueue();
             // two of the three commits go to the renamed table
-            String expected = "location\ttemperature\ttimestamp\n" +
+            final String expected = "location\ttemperature\ttimestamp\n" +
                     "west1\t10.0\t2016-06-13T17:43:50.100400Z\n" +
                     "west2\t20.0\t2016-06-13T17:43:50.100500Z\n" +
                     "east3\t30.0\t2016-06-13T17:43:50.100600Z\n" +
                     "west4\t40.0\t2016-06-13T17:43:50.100700Z\n" +
                     "south8\t80.0\t2016-06-13T17:43:50.101000Z\n";
-            assertTable(expected, meteorology);
+
+            assertEventually(
+                    () -> {
+                        drainWalQueue();
+                        assertTable(expected, meteorology);
+                    },
+                    15
+            );
 
             // last commit goes to the recreated table
-            expected = "location\ttemperature\ttimestamp\n" +
+            final String expected2 = "location\ttemperature\ttimestamp\n" +
                     "west5\t50.0\t2016-06-13T17:43:50.100800Z\n" +
                     "east6\t60.0\t2016-06-13T17:43:50.100900Z\n" +
                     "south7\t70.0\t2016-06-13T17:43:50.101000Z\n";
-            assertTable(expected, weather);
+
+            assertEventually(
+                    () -> {
+                        drainWalQueue();
+                        assertTable(expected2, weather);
+                    },
+                    15
+            );
 
         }, false, 250);
     }
@@ -1441,7 +1456,7 @@ public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
             sendLinger(lineData.toString(), tableName);
             doneLatch.await();
 
-            TestUtils.assertEventually(() -> {
+            assertEventually(() -> {
                 mayDrainWalQueue();
 
                 CharSequenceIntHashMap symbolCounts = new CharSequenceIntHashMap();

--- a/core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpReceiverTest.java
+++ b/core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpReceiverTest.java
@@ -746,7 +746,6 @@ public class LineTcpReceiverTest extends AbstractLineTcpReceiverTest {
                                 --count == 0
                 ) {
                     renameTable(weather, meteorology);
-                    Os.sleep(100);
                 }
                 return super.openRW(name, opts);
             }

